### PR TITLE
Revise appearance for feature-state shortcode

### PR DESCRIPTION
**Pull request by sftim**
Change how feature-state shortcodes look. The new look makes them stand out more, and makes it easier to use a different style in the future. We can build on this to, eg, color alpha and beta feature state notices with a special outline.

This PR also deletes a legacy CSS file.

/area web-development

[Current example](https://k8s.io/docs/reference/access-authn-authz/authentication/#bootstrap-tokens) | [Preview](https://deploy-preview-45981--kubernetes-io-main-staging.netlify.app/docs/reference/access-authn-authz/authentication/#bootstrap-tokens)

### CODE REVIEW

File: layouts/shortcodes/feature-state.html [hunk 1]
```diff
@@ -31,19 +31,19 @@
...
           {{- $currentStage := index $featureGateFile.Params.stages (sub (len $featureGateFile.Params.stages) 1) -}}
-          {{- with $currentStage -}}  
+          {{- with $currentStage -}}
            
           <!-- Display feature state information  -->
-          <div class="my-2 feature-state-notice feature-{{ .stage }}" title="{{ printf "%s %s" (T "feature_state_feature_gate_tooltip") $feature_gate_name }}">
-            <b>{{ T "feature_state" }}</b> <code>{{T "feature_state_kubernetes_label" }} v{{ .fromVersion }} [{{ .stage }}]</code>
-          </div>       
+          <div class="feature-state-notice feature-{{ .stage }}" title="{{ printf "%s %s" (T "feature_state_feature_gate_tooltip") $feature_gate_name }}">
+            <span class="feature-gate-name">{{ T "feature_state" }}</span> <code>{{T "feature_state_kubernetes_label" }} v{{ .fromVersion }} [{{ .stage }}]</code>
```
Comment (dipesh-rawat) [inline layouts/shortcodes/feature-state.html:42]: Nice work overall!

There's a slight issue with the class name assigned to the `span` element, causing some inconsistent rendering incase of feature state retrieved from description file. 

[Preview Link highlighting the problem (click here)](https://deploy-preview-45981--kubernetes-io-main-staging.netlify.app/docs/contribute/style/hugo-shortcodes/#feature-state-retrieval-from-description-file)


 
```suggestion
            <span class="feature-state-name">{{ T "feature_state" }}</span> <code>{{T "feature_state_kubernetes_label" }} v{{ .fromVersion }} [{{ .stage }}]</code>
```

**Review by dipesh-rawat — COMMENTED** [review]
💭 While working on feature gates, I noticed an existing CSS file **([feature-states.css](https://github.com/kubernetes/website/blob/main/static/css/feature-states.css))** related to applying styling to feature states. Not sure if it's meant to be used for this purpose or if it's now obsolete. It might be worth investigating its current relevance particularly when building upon this pull request.

File: static/css/feature-states.css [hunk 2]
```diff
@@ -1,39 +0,0 @@
...
-
-.alpha {    
-    color: #8a6d3b;
-    border-color: #faebcc;
-}
-
-.deprecated {    
-    color: #a94442;
-    border-color: #ebccd1;
-}
```
Comment (sftim) [inline static/css/feature-states.css:30]: We could bring these (currently unused) colors back some time, if we'd like to.

**Review by dipesh-rawat — COMMENTED** [review]
Nice improvement, looks good to me!

/lgtm
`/approve` (Unofficially giving the nod of approval; this officially need an approval from **sig-docs-website-owners**)

**Pull request merged**